exercises/triangle/src/lib.rs

```diff
@@ -1 +1,19 @@
+pub struct Triangle;
 
+impl Triangle {
+    pub fn build(sides: [u64; 3]) -> Result<Triangle, ()> {
```

Comment:
I'm aware that the signature isn't your fault, but I don't like the practice of returning `Result<T, ()>`; that should idiomatically be an `Option<T>` instead. 

If you're willing, I would appreciate it if you would update the exercise to use an Option here. If not, feel free to ignore.

Reply:
Will do